dockerv2.go

```diff
@@ -14,6 +16,10 @@ var dockerRegexes = map[string]*regexp.Regexp{
 
 func redirectDockerv2(w http.ResponseWriter, r *http.Request, rec record) error {
 	path := r.URL.Path
+	if !strings.Contains(path, "/v2") {
+		log.Printf("<%s> [txtdirect]: unrecognized path for dockerv2: %s", time.Now().Format(logFormat), path)
+		fallback(w, r, rec.Root, http.StatusPermanentRedirect, Config{})
```

Comment:
Do we agree that we should use the fallback mechanism here and if rec.Root is provided always redirect there?
This is different than the other fallbacks. Also it seemed you disliked the idea of rec.Root being used as a general fallback.
Other options is use an additional fallback value, redirect to rec.To (doesn't perfectly work with base redirects such as "gcr.io/".

/ -> rec.Root
/ -> !rec.Root -> fallback
/random -> ?